### PR TITLE
Update AdapterVersion log message

### LINKED ISSUE #3726: AdapterVersion log message
### Contributions

- [X] I have read the contribution docs, and understand what's expected of me.

### Link to the page on docs.getdbt.com requiring updates

https://docs.getdbt.com/docs/dbt-versions/core#how-we-version-adapter-plugins

### What part(s) of the page would you like to see updated?

- Use a newer version
- Include the "registered adapter" version as well

```
$ dbt --version
Core:
  - installed: 1.5.3
  - latest:    1.5.3 - Up to date!

Plugins:
  - snowflake: 1.5.2 - Up to date!
```

### Additional information

- https://github.com/dbt-labs/dbt-core/issues/7038
- https://github.com/dbt-labs/dbt-core/pull/7862
- Included in v1.5.2 + v1.6.0

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Updating the AdapterVersion log message in this section:

https://docs.getdbt.com/docs/dbt-versions/core#how-we-version-adapter-plugins 

Resolves #3726 

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Needs review from product team

### CODE REVIEW

File: website/docs/docs/dbt-versions/core-versions.md [hunk 1]
```diff
@@ -96,12 +96,11 @@ That means that patch version numbers will likely differ between `dbt-core` and
...
-   latest version: 1.2.3
-
-Up to date!
+Core:
+  - installed: 1.5.3
+  - latest:    1.5.3 - Up to date!
 
 Plugins:
-  - snowflake: 1.2.0 - Up to date!
+  - snowflake: 1.5.2 - Up to date!
```
Comment (dbeatty10) [inline website/docs/docs/dbt-versions/core-versions.md]: Could we go all the way up to 1.6.0?

Comment (nghi-ly) [reply website/docs/docs/dbt-versions/core-versions.md]: sure thing! i can update the `Core` section to 1.6.0

File: website/docs/docs/dbt-versions/core-versions.md [hunk 2]
```diff
@@ -96,12 +96,11 @@ That means that patch version numbers will likely differ between `dbt-core` and
 For example, you may find you're using `dbt-core==1.2.3` with `dbt-snowflake==1.2.0`. It is critical that you're using the latest patch available for both core and the adapter (v1.2.x). Use the `dbt --version` command to see which versions you have installed:
 ```
 $ dbt --version
-installed version: 1.2.3
-   latest version: 1.2.3
-
-Up to date!
+Core:
+  - installed: 1.5.3
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/core-versions.md]:
```suggestion
  - installed: 1.6.0
```

File: website/docs/docs/dbt-versions/core-versions.md [hunk 3]
```diff
@@ -96,12 +96,11 @@ That means that patch version numbers will likely differ between `dbt-core` and
 For example, you may find you're using `dbt-core==1.2.3` with `dbt-snowflake==1.2.0`. It is critical that you're using the latest patch available for both core and the adapter (v1.2.x). Use the `dbt --version` command to see which versions you have installed:
 ```
 $ dbt --version
-installed version: 1.2.3
-   latest version: 1.2.3
-
-Up to date!
+Core:
+  - installed: 1.6.0
+  - latest:    1.5.3 - Up to date!
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/core-versions.md]:
```suggestion
  - latest:    1.6.0 - Up to date!
```

File: website/docs/docs/dbt-versions/core-versions.md [hunk 4]
```diff
@@ -96,12 +96,11 @@ That means that patch version numbers will likely differ between `dbt-core` and
...
-   latest version: 1.2.3
-
-Up to date!
+Core:
+  - installed: 1.6.0
+  - latest:    1.6.0 - Up to date!
 
 Plugins:
-  - snowflake: 1.2.0 - Up to date!
+  - snowflake: 1.5.2 - Up to date!
```
Comment (dbeatty10) [inline website/docs/docs/dbt-versions/core-versions.md]: Just to keep in aligned with dbt-core above:
```suggestion
  - snowflake: 1.6.0 - Up to date!
```

**Comment by nghi-ly** [comment]
@dbeatty10 : folded in your review comment. can i get a re-review? tysm!

**Review by dbeatty10 — COMMENTED** [review]
Jerco was hoping for these updates to include an example of what the "registered adapter" version introduced in https://github.com/dbt-labs/dbt-core/pull/7862 looks like too.

Here's an example of how the registered adapter version displays within `logs/dbt.log`:

```
[0m13:13:48.572182 [info ] [MainThread]: Registered adapter: snowflake=1.6.0
```

File: website/docs/docs/dbt-versions/core-versions.md [hunk 5]
```diff
@@ -93,15 +93,20 @@ When you use dbt, you use a combination of `dbt-core` and an adapter plugin spec
...
+Core:
+  - installed: 1.6.0
+  - latest:    1.6.0 - Up to date!
 
 Plugins:
-  - snowflake: 1.2.0 - Up to date!
+  - snowflake: 1.6.0 - Up to date!
+```
+
+When running `dbt debug`, you can see which version of the registered adapter version that's being invoked. Below is an example of the message in the `logs/dbt.log` file:
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/core-versions.md]:
```suggestion
When running `dbt debug`, you can see which version of the registered adapter that's being invoked. Below is an example of the message in the `logs/dbt.log` file: 
```

**Comment by nghi-ly** [comment]
thanks for flagging, @dbeatty10 ! 

added content for the "registered adapter version". not sure if i did it right tho. can you take a look? tysm!

File: website/docs/docs/dbt-versions/core-versions.md [hunk 6]
```diff
@@ -93,15 +93,20 @@ When you use dbt, you use a combination of `dbt-core` and an adapter plugin spec
...
+Core:
+  - installed: 1.6.0
+  - latest:    1.6.0 - Up to date!
 
 Plugins:
-  - snowflake: 1.2.0 - Up to date!
+  - snowflake: 1.6.0 - Up to date!
+```
+
+When running `dbt debug`, you can see which version of the registered adapter that's being invoked. Below is an example of the message in the `logs/dbt.log` file:
```
Comment (dbeatty10) [inline website/docs/docs/dbt-versions/core-versions.md]: The version for the registered adapter should be visible in the [logs](https://docs.getdbt.com/reference/global-configs/logs) for _all_ dbt commands (not just `dbt debug`).

So maybe something like this?

```suggestion
You can see which version of the registered adapter that's being invoked in the [logs](/reference/global-configs/logs). Below is an example of the message in the `logs/dbt.log` file: 
```

**Review by dbeatty10 — APPROVED** [review]
:shipit:

**Pull request merged**